enterprise-cfengine-guide/settings.markdown

```diff
@@ -49,24 +49,68 @@ users, including their name, role, and password.
 
 ![Role Management](Settings-role.png)
 
-Roles are used to lmit access. Roles can limit access for reporting on
-hosts that have or do not have spcecific classes, and they can be used
-to limit which sketches from Design Center a user is authorized to
-use.
+Roles limit access to host data and access to shared assets like saved reports
+and dashboards.
 
-For example if you want to limit a users ability to report only on
-hosts in the "North American Data Center" you could setup a role that
-includes only the `location_nadc` class.
+Roles limit access to which hosts can be seen based on the classes reported by
+the host. For example if you want to limit a users ability to report only on
+hosts in the "North American Data Center" you could setup a role that includes
+only the `location_nadc` class.
 
-If you have multiple roles assigned to a user, the user can access
-only hosts that match all of their role restrictions. For example, if
-you have the admin role and a role that matches zero hosts, the user
-will not see any hosts in Mission Portal.
+When multiple roles assigned to a user, the user can access only resources that
```

Comment:
When multiple roles are assigned... Just missing the word are

Reply:
@nickanderson I made another PR as an example of what I'd like to see. I also think I fixed some other formatting inconsistencies in the example.

https://github.com/cfengine/documentation/pull/1829 (master)